import_export_batches/models.py

```diff
@@ -1417,6 +1446,10 @@ class BatchRowActionOffice(models.Model):
                                                  max_length=255, null=True, blank=True)
     # "Yes" or "No" depending on whether this a contest being held outside the normal election cycle.
     special = models.CharField(verbose_name="google civic primary party", max_length=255, null=True, blank=True)
+    ctcl_uuid = models.CharField(verbose_name="ctcl uuid", max_length=80, null=True, blank=True)
+    office_description = models.CharField(verbose_name="office description", max_length=255, null=True, blank=True)
+    office_is_partisan = models.BooleanField(verbose_name='office is_partisan', default=False)
+    status = models.CharField(verbose_name="Batch Row Action Office Status", max_length=80, null=True, blank=True)
 
```

Comment:
Hi Neelam, please make the text in verbose_name all lower case in the future. Thank you!